### DISCUSSION TITLE
Tooltip not showing well after inverted range axis on vertical plot

### DISCUSSION THREAD

**Discussion by Belegosth** [jfree/jfreechart#310] [Q&A] [answered]
Hello, hope you're getting well.

After looking on "Tooltips for XYLineAndShapeRenderer not working in HORIZONTAL plot orientation #48"

I still have the problem.

My plot orientation is vertical and inverted the range axis like so : 
plot.getRangeAxis().setInverted(true);

but the tooltip is still showing like it's not inverted. 

I Run the 1.5.2.

Thank for any advice

**Comment by trashgod** [comment]
I tried the example cited in issue #48; after [updating](https://github.com/jfree/jfreechart#migration), I could not reproduce the problem with v1.5.3.

**Comment by Belegosth** [comment, marked as answer]
Yeah that a "deep" error from my side ... Sorry for inconvenience =/
